_posts/2025-04-28-secure-mcp-sse-server.adoc

```diff
@@ -314,7 +315,16 @@ Launch https://modelcontextprotocol.io/docs/tools/inspector[MCP inspector]:
 npx @modelcontextprotocol/inspector
 ----
 
-Paste the copied GitHub access token to the `Bearer Token` field and connect to the Quarkus MCP SSE server:
+[NOTE]
+====
+Ensure that you have https://github.com/modelcontextprotocol/inspector/releases/tag/0.6.0[modelcontextprotocol/inspector] version 0.6.0 or later installed as it adds support for specifying bearer token authentication.
```

Comment:
FYI, I had `0.10.6` or so and when I run it today it offered me an option to update and did update to 0.11.0, not sure we need to link to the rather old version as `npx @modelcontextprotocol/inspector` seems to pick up the latest anyway... please check

Reply:
This is the version where bearer auth is supported. Anything afterward is fine

Reply:
@sabre1041 Sure, my point is, the blog advises to start with the `npx @modelcontextprotocol/inspector` command, and it ensures the latest MCP inspector is loaded, can you please confirm it ?

Reply:
Never mind, let's keep this advice, you are right